client/Sample/freerdp.c

```diff
@@ -146,7 +146,9 @@ static void* tf_client_thread_proc(freerdp* instance)
 
 		if (!freerdp_check_event_handles(instance->context))
 		{
-			WLog_ERR(TAG, "Failed to check FreeRDP event handles");
+			if (freerdp_get_last_error(instance->context) == FREERDP_ERROR_SUCCESS)
```

Comment:
Think that should be !=

Reply:
No, only print something if no error has been set (which already printed a more detailed error message)

Reply:
Ah sorry - thought the "Failed to check FreeRDP event handles" message in case of ERROR_SUCCESS was the problem ... alright then it LGTM